test/test-remote/test_ui.ts

```diff
@@ -363,4 +363,34 @@ suite("test_ui_with_headless_browser", function () {
       await sleep(10);
     }
   });
+
+  test("view_ring_health", async function () {
+    // Same consideration as above: this test is here in this module for now
+    // just because it's easy to use the authentication state after actual
+    // UI-based login.
+    assert(COOKIES_AFTER_LOGIN);
+    const page = await BROWSER.contexts()[0].newPage();
+    
+    await page.goto(CLUSTER_BASE_URL + "/cluster/health/metrics/ingester");
+    assert(await page.isVisible("text=Cortex Ring Health"));
```

Comment:
there might be better ways to assert that this integration is working - but I cannot use the IDs, as some of them are UUIDs. 
Any suggestion, or is this okay for now?